lib/services/local/users.go

```diff
@@ -62,7 +63,8 @@ var GlobalSessionDataMaxEntries = 5000 // arbitrary
 // user accounts as well
 type IdentityService struct {
 	backend.Backend
-	log logrus.FieldLogger
+	log        logrus.FieldLogger
+	bcryptCost int
```

Comment:
Suggestion: Explicitly set the default cost on NewIdentityService?

I realize bcrypt will default `0` to bcrypt.DefaultCost, but I think being more explicit is likely to cause less confusion.

Reply:
Good point.